src/test/data/entity/fullAdmission.ts

```diff
@@ -0,0 +1,257 @@
+import { MomentFactory } from 'shared/momentFactory'
+import { FormaliseOption } from 'claims/models/claimant-response/formaliseOption'
+import { ClaimantResponseType } from 'claims/models/claimant-response/claimantResponseType'
+import { PaymentOption } from 'claims/models/paymentOption'
+import { PaymentSchedule } from 'claims/models/response/core/paymentSchedule'
+
+export const immediatePaymentIntention = {
+  paymentDate: MomentFactory.currentDate().add(5, 'days'),
+  paymentOption: PaymentOption.IMMEDIATELY
+}
+
+export const setByDatePaymentIntention = {
+  paymentDate: MomentFactory.currentDate().add(30, 'days'),
+  paymentOption: PaymentOption.BY_SPECIFIED_DATE
+}
+
+export const setByDatePaymentIntentionPastDeadline = {
+  paymentDate: MomentFactory.currentDate().subtract(1, 'days'),
+  paymentOption: PaymentOption.BY_SPECIFIED_DATE
+}
+
+export const installmentsPaymentIntentionPastDeadline = {
```

Comment:
"installment" is US English - please find/replace to "instalment"

Reply:
Thanks, updated